modules/apps/server/server-admin-web/src/main/java/com/liferay/server/admin/web/internal/portlet/action/EditServerMVCActionCommand.java

```diff
@@ -214,7 +214,7 @@ else if (cmd.equals("dlDeletePreviews")) {
 			DLPreviewableProcessor.deleteFiles();
 		}
 		else if (cmd.equals("dlGeneratePDFPreviews")) {
-			_pdfProcessor.generatePDFPreviews();
+			_pdfProcessor.generatePreviews();
 		}
```

Comment:
Should now the option "dlGeneratePDFPreviews" be "dlGeneratePreviews"? Or may the method names should all have changed to `generatePdfPreviews()` instead of `generatePreviews` (even `PDFPreviewBackgroundTaskExecutor._generatePreviews`) ?